### PR TITLE
Shuckle Imagemaker fix

### PR DESCRIPTION
**Pull request by Shidouuu**
Everything should work perfectly now.

Also, a minor update to requirements.txt for modules that I've used and have worked, minus simplejson since the newer version was just a bug fix.

Additionally, I changed some if's to elif's in Shuckbot.py.

### CODE REVIEW

File: modules/Shuckbot.py [hunk 1]
```diff
@@ -199,19 +199,21 @@ async def on_message(message):
         elif content.lower().startswith(("tom", "tomscott")):
             await imagefun.tom_imagemaker(message)
 
-    if message.clean_content.lower() == "b" or message.clean_content.lower() == "n":
+        elif content.lower().startswith(("shuck")):
+            await imagefun.shuckle_imagemaker(message)
```
Comment (Shidouuu) [inline modules/Shuckbot.py:203]: Oops, forgot to press enter. Still works though

File: modules/Shuckbot.py [hunk 2]
```diff
@@ -199,19 +199,21 @@ async def on_message(message):
...
+    elif message.clean_content.lower() == "b" or message.clean_content.lower() == "n":
```
Comment (Shidouuu) [inline modules/Shuckbot.py]: Oops, forgot to press enter. Still works though

**Comment by will-j-wright** [comment]
Make these edits in the base Shuckbot.py file not modules/Shuckbot.py—that file doesn’t exist in production and was erroneously placed in the repo

**Comment by Shidouuu** [comment]
Whoops, didn't notice that. Changes have been made.

**Comment by Shidouuu** [comment]
I added Hirtsifier functionality: `;hirtsifier <text>`

Also, I changed the based elif statement back to an if and moved it to the bottom so it will still activate when using another command, ex. `@ShuckBot based`

Edit: Omitted some stuff accidentally so I pushed another commit

**Pull request closed without merging**